feat(curriculum): add merge sort project to Python curriculum (#52469)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-data-structures-by-building-the-merge-sort-algorithm/656710d1e0ec62253426db24.md

---
id: 656710d1e0ec62253426db24
title: Step 18
challengeType: 20
dashedName: step-18
---

# --description--

> The first `while` loop we created compares one element from `left_part`with another in `right_part`, then adds the smaller element to the main `array` list.

Reviewer: ```suggestion
The `while` loop you created compares one element from `left_part`with another in `right_part`, then adds the smaller element to the main `array` list.
```
